### DISCUSSION TITLE
How to deploy node application to local or remote server

### DISCUSSION THREAD

**Discussion by ghamandipp** [concourse/concourse#8207] [General]
How to deploy node application to local or remote server?

**Comment by taylorsilva** [comment]
How ever you want! I usually like to figure out how to do things on my local computer first and then port it over to a script in Concourse.

So maybe you want to ask "how do I deploy a node app to a remote server?". Once you have an answer to that then add "with concourse" to your question. The answer will probably be more obvious once the first question is answered.

Sorry for lack of specifics. First, I don't know how node app's are deployed, and there's also 10 million way to deploy app FOO in language BAR on platform CLOUD.
